include/cantera/zeroD/ConstPressureMoleReactor.h

```diff
@@ -26,18 +26,26 @@ class ConstPressureMoleReactor : public MoleReactor
         return "ConstPressureMoleReactor";
     };
 
-    size_t componentIndex(const string& nm) const override;
-
-    string componentName(size_t k) override;
-
     void getState(double* y) override;
 
     void initialize(double t0=0.0) override;
 
     void eval(double t, double* LHS, double* RHS) override;
 
+    vector<size_t> steadyConstraints() const override {
+        throw CanteraError("ConstPressureMoleReactor::steadyConstraints",
+            "ConstPressureMoleReactor is not currently compatible with the steady-state"
+            " solver.");
```

Comment:
Should we create an enhancements issue to link to for people to indicate their support for this work?

Reply:
Done. See https://github.com/Cantera/enhancements/issues/234.